.github/workflows/release.yml

```
@@ -0,0 +1,50 @@
+name: Release
+
+on:
+    push:
+      branches:
+        - main
+        - next
+
+permissions:
+  contents: read
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 4 but found 2

(indentation)